apps/webpack.js

```diff
@@ -90,8 +97,71 @@ var karmaConfig = _.extend({}, config, {
   ]
 });
 
+/**
+ * Generate the appropriate webpack config based off of our base config and
+ * some input options
+ * @param {object} options
+ * @param {string} options.output
+ * @param {string[]} options.entries - list of input source files
+ * @param {bool} options.minify
+ * @param {bool} options.watch
+ * @param {string} options.piskelDevMode
+ */
+function create(options) {
+  var outputDir = options.output;
```

Comment:
I guess that depends a little bit on what the structure of our outputs ends up looking like after we've merged in code-studio. Probably though the right answer is to not have it be parameterized to start, and then change it if we decide that's what we want/need when building code-studio